satellite/repair/repair_test.go

```diff
@@ -50,29 +50,29 @@ func TestDataRepair(t *testing.T) {
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
 		// first, upload some remote data
 		uplinkPeer := planet.Uplinks[0]
-		satellite := planet.Satellites[0]
+		sat := planet.Satellites[0]
```

Comment:
Please, revert the refactoring done in this file. It goes against code clarity.

Reply:
@kaloyan-raev Where is that agains code clarity?
We are shadowing the imports with those var names. That is way worse imo?

Reply:
There are such examples is all over our codebase. It's not an issue unless the compiler complains. You may bring this up to the #goalng channel in Slack, if you want to discuss this general.

The refactoring in this PR makes the code less readable and bloats the review of the PR itself.

Reply:
My IDE complains. I am changing this back and make a seperate PR to fix it across the entire codebase.

Reply:
Thanks! Just make sure that you discuss with the team if we really want this refactoring before spending the time on it.